golden_response/staging/get_places_in/county_zip.json

```diff
@@ -1,59 +1,230 @@
 [
-  { "dcid": "geoId/06085", "place": "zip/94022" },
-  { "dcid": "geoId/06085", "place": "zip/94024" },
-  { "dcid": "geoId/06085", "place": "zip/94040" },
-  { "dcid": "geoId/06085", "place": "zip/94041" },
-  { "dcid": "geoId/06085", "place": "zip/94043" },
-  { "dcid": "geoId/06085", "place": "zip/94085" },
-  { "dcid": "geoId/06085", "place": "zip/94086" },
-  { "dcid": "geoId/06085", "place": "zip/94087" },
-  { "dcid": "geoId/06085", "place": "zip/94089" },
-  { "dcid": "geoId/06085", "place": "zip/94301" },
-  { "dcid": "geoId/06085", "place": "zip/94304" },
-  { "dcid": "geoId/06085", "place": "zip/94305" },
-  { "dcid": "geoId/06085", "place": "zip/94306" },
-  { "dcid": "geoId/06085", "place": "zip/95002" },
-  { "dcid": "geoId/06085", "place": "zip/95008" },
-  { "dcid": "geoId/06085", "place": "zip/95013" },
-  { "dcid": "geoId/06085", "place": "zip/95014" },
-  { "dcid": "geoId/06085", "place": "zip/95020" },
-  { "dcid": "geoId/06085", "place": "zip/95030" },
-  { "dcid": "geoId/06085", "place": "zip/95032" },
-  { "dcid": "geoId/06085", "place": "zip/95035" },
-  { "dcid": "geoId/06085", "place": "zip/95037" },
-  { "dcid": "geoId/06085", "place": "zip/95046" },
-  { "dcid": "geoId/06085", "place": "zip/95050" },
-  { "dcid": "geoId/06085", "place": "zip/95051" },
-  { "dcid": "geoId/06085", "place": "zip/95053" },
-  { "dcid": "geoId/06085", "place": "zip/95054" },
-  { "dcid": "geoId/06085", "place": "zip/95070" },
-  { "dcid": "geoId/06085", "place": "zip/95110" },
-  { "dcid": "geoId/06085", "place": "zip/95111" },
-  { "dcid": "geoId/06085", "place": "zip/95112" },
-  { "dcid": "geoId/06085", "place": "zip/95113" },
-  { "dcid": "geoId/06085", "place": "zip/95116" },
-  { "dcid": "geoId/06085", "place": "zip/95117" },
-  { "dcid": "geoId/06085", "place": "zip/95118" },
-  { "dcid": "geoId/06085", "place": "zip/95119" },
-  { "dcid": "geoId/06085", "place": "zip/95120" },
-  { "dcid": "geoId/06085", "place": "zip/95121" },
-  { "dcid": "geoId/06085", "place": "zip/95122" },
-  { "dcid": "geoId/06085", "place": "zip/95123" },
-  { "dcid": "geoId/06085", "place": "zip/95124" },
-  { "dcid": "geoId/06085", "place": "zip/95125" },
-  { "dcid": "geoId/06085", "place": "zip/95126" },
-  { "dcid": "geoId/06085", "place": "zip/95127" },
-  { "dcid": "geoId/06085", "place": "zip/95128" },
-  { "dcid": "geoId/06085", "place": "zip/95129" },
-  { "dcid": "geoId/06085", "place": "zip/95130" },
-  { "dcid": "geoId/06085", "place": "zip/95131" },
-  { "dcid": "geoId/06085", "place": "zip/95132" },
-  { "dcid": "geoId/06085", "place": "zip/95133" },
-  { "dcid": "geoId/06085", "place": "zip/95134" },
-  { "dcid": "geoId/06085", "place": "zip/95135" },
-  { "dcid": "geoId/06085", "place": "zip/95136" },
-  { "dcid": "geoId/06085", "place": "zip/95138" },
-  { "dcid": "geoId/06085", "place": "zip/95139" },
-  { "dcid": "geoId/06085", "place": "zip/95140" },
-  { "dcid": "geoId/06085", "place": "zip/95148" }
+  {
```

Comment:
I thought we were done with formatting diffs?

Reply:
turns out the place_in api was not covered by the update_golden() script, and this time there is a real diff which caught this up. 

good thing is the real diff is a potential bug.

Reply:
https://github.com/datacommonsorg/mixer/pull/305/files#diff-089444bf72fece8a66ffe0762b823e0fR77-R81

Reply:
I don't think that link works